common/djangoapps/student/models.py

```diff
@@ -332,6 +337,57 @@ def user_profile_pre_save_callback(sender, **kwargs):
         user_profile.profile_image_uploaded_at = None
 
 
+@receiver(post_save, sender=UserProfile)
+def user_profile_post_save_callback(sender, **kwargs):
+    """
+    Emit analytics events after saving the
+    """
+    user_profile = kwargs['instance']
+    # pylint: disable=protected-access
+    emit_field_changed_events(user_profile, user_profile.user, sender._meta.db_table)
+
+
+def emit_field_changed_events(instance, user, db_table):
+    """ For the given model instance, save the fields that have changed since the last save.
+
+    Requires that the Model uses 'model_utils.FieldTracker' and has assigned it to 'tracker'.
+
+    Args:
+        instance (Model instance): the model instance that is being saved
+        user (User): the user that this instance is associated with
+        db_table (str): the name of the table that we're modifying
+
+    Returns:
+        None
+    """
+    excluded_fields = ['meta', 'password']
```

Comment:
We actually are going to want to emit password change events, though we don't want to send the old and new values. Perhaps the thought is that special-case code appears elsewhere.